packages/toolbox/src/commands/chainlink/aggregatorsUtils.ts

```diff
@@ -29,35 +17,34 @@ export type Aggregator = {
 const feedMap: Partial<
   Record<CurrencyTypes.EvmChainName, [chainKey: string, networkName: string]>
 > = {
-  mainnet: ['ethereum', 'Ethereum Mainnet'],
-  goerli: ['ethereum', 'Goerli Testnet'],
-  rinkeby: ['ethereum', 'Rinkeby Testnet'],
-  fantom: ['fantom', 'Fantom Mainnet'],
-  matic: ['polygon', 'Polygon Mainnet'],
-  xdai: ['gnosis-chain', 'Gnosis Chain Mainnet'],
-  bsc: ['bnb-chain', 'BNB Chain Mainnet'],
-  avalanche: ['avalanche', 'Avalanche Mainnet'],
-  optimism: ['optimism', 'Optimism Mainnet'],
-  'arbitrum-one': ['arbitrum', 'Arbitrum Mainnet'],
-  moonbeam: ['moonbeam', 'Moonbeam Mainnet'],
+  mainnet: ['mainnet', 'Ethereum Mainnet'],
+  goerli: ['goerli', 'Goerli Testnet'],
+  fantom: ['fantom-mainnet', 'Fantom Mainnet'],
+  matic: ['matic-mainnet', 'Polygon Mainnet'],
+  xdai: ['xdai-mainnet', 'Gnosis Chain Mainnet'],
+  bsc: ['bsc-mainnet', 'BNB Chain Mainnet'],
+  avalanche: ['avalanche-mainnet', 'Avalanche Mainnet'],
+  optimism: ['ethereum-mainnet-optimism-1', 'Optimism Mainnet'],
+  'arbitrum-one': ['ethereum-mainnet-arbitrum-1', 'Arbitrum Mainnet'],
+  moonbeam: ['polkadot-mainnet-moonbeam', 'Moonbeam Mainnet'],
 };
 
-export const getAllAggregators = async (network: CurrencyTypes.EvmChainName): Promise<Proxy[]> => {
+export const getAllAggregators = async (network: CurrencyTypes.EvmChainName): Promise<Feed[]> => {
   const [feedName, networkName] = feedMap[network] || [];
   if (!feedName || !networkName) {
     throw new Error(
       `network ${network} not supported by feed provider. Is it supported by Chainlink?`,
     );
   }
-  const { data } = await axios.get<Record<string, Feed>>(
-    'https://cl-docs-addresses.web.app/addresses.json',
+
+  const { data: feeds } = await axios.get<Feed[]>(
+    `https://reference-data-directory.vercel.app/feeds-${feedName}.json`,
   );
```

Comment:
In this file, I changed the API for fetching feeds. I don't know where the previous URL came from, I can't find any doc on it, and it looked outdated. For instance, there was no Arbitrum feed. The new URL is more or less the official "Chainlink feed API": it is the API Chainlink uses on their feed list page [https://docs.chain.link/data-feeds/price-feeds/addresses](https://docs.chain.link/data-feeds/price-feeds/addresses)

Reply:
the previous address came from retro engineering their page :D
nothing documented